symforce/codegen/codegen_config.py

```diff
@@ -77,3 +77,26 @@ def format_data_accessor(prefix: str, index: int) -> str:
         Format data for accessing a data array in code.
         """
         return f"{prefix}.data[{index}]"
+
+    # TODO: Move this into code printer.
+    @staticmethod
+    def format_matrix_subscript_accessor(key: str, i: int, j: int) -> str:
+        """
+        Format accessing a matrix element (i, j) in code.
+        Args:
+            key (str): Name of the variable being accessed.
+            i (int): Row
+            j (int): Column
+        """
+        return f"{key}({i}, {j})"
+
+    @abstractmethod
```

Comment:
Does this conceptually mean each config is required to define this? Since the default of `pass` is good enough, perhaps this shouldn't be an abstractmethod

Reply:
Yes, this is correct. I will remove the @abstractmethod decorator.